static-build/pages/index/index.tsx

```diff
@@ -114,112 +115,127 @@ const IndexPage: FunctionalComponent<Props> = ({ tests }: Props) => {
-                  popularity; to cover the most surface area. We are actively
-                  aggregating feedback for the next set of tools and tests.
-                </p>
+            <section id="data">
+              <div class={$sidebarLayout}>
+                <div></div>
+                <div class={$gettingStarted}>
+                  <h3>Information</h3>
+                  <p>
+                    Below you'll get to see how your favorite, or new to be
+                    favorite, tool is handling our industry best practice test
+                    suite.
+                  </p>
+                  <p class={$well}>Each tool scores as follows:</p>
+                  <Legend />
+                  <h4>The Tools</h4>
+                  <p>
+                    We chose browserify, parcel, rollup & webpack first out of
+                    popularity; to cover the most surface area. We are actively
+                    aggregating feedback for the next set of tools and tests.
+                  </p>
+                </div>
               </div>
-            </div>
-          </section>
+            </section>
+          </div>
 
           <ToolNav />
 
-          <section class={`${$overview} ${$overviewGrid}`}>
-            <a href="#overview">
-              <h2 id="overview" class={`${$overviewHeader} ${$sectionHeader}`}>
-                <span class={$sectionHashtag}>#</span>
-                Overview
-              </h2>
-            </a>
-            <DataGrid tests={tests} basePath="/" collectionTitle="Overview" />
-          </section>
-
-          {Object.entries(tests).map(([testDir, collection]) => {
-            const sectionId = collection.meta.title
-              .replace(/\s/g, '-')
-              .toLowerCase();
-
-            return (
-              <section>
-                <h3 class={$sectionHeader} id={sectionId}>
-                  <a class={$sectionHashtag} href={`#${sectionId}`}>
-                    #
-                  </a>
-                  <a class={$sectionTitle} href={`${testDir}/`}>
-                    {collection.meta.title}
-                  </a>
-                </h3>
+          <div class={$overviewContainer}>
+            <section class={`${$contentContainer} ${$overviewContent}`}>
+              <div class={$overviewGrid}>
+                <a href="#overview">
+                  <h2
+                    id="overview"
+                    class={`${$overviewHeader} ${$sectionHeader}`}
+                  >
+                    <span class={$sectionHashtag}>#</span>
+                    Overview
+                  </h2>
+                </a>
                 <DataGrid
-                  tests={collection.subTests}
-                  basePath={`${testDir}/`}
-                  collectionTitle={collection.meta.title}
+                  tests={tests}
+                  basePath="/"
+                  collectionTitle="Overview"
```

Comment:
The diff is getting confused here. I didn't change this, it's mismatching with the DataGrid further up.